packages/shared/args.js

```diff
@@ -0,0 +1,30 @@
+function args(args) {
+    const parsedArgs = {
+        named: { },
+        positional: []
+    };
+    const [...rest] = args;
+
+    rest.forEach((arg, index) => {
+        if (arg.indexOf("-") != -1) {
+            const name = getArgName(arg)
+            parsedArgs.named[name] = rest[index + 1];
+            rest.shift();
+        } else {
+            parsedArgs.positional.push(arg);
+        }
+    });
+
+    return parsedArgs;
+}
+
+function getArgName(arg) {
+    if (arg[1] === '-') return arg.slice(2, arg.length);
```

Comment:
You should be able to run `yarn format` from the root to auto format the files according to project conventions, just making sure you are aware ;)